silverback/runner.py

```diff
@@ -46,6 +46,15 @@ async def _event_task(
         """
 
     async def run(self):
+        """
+        Run the task broker client for the assembled `SilverbackApp` application.
+
+        Will listen for events against the connected provider (using `ManagerAccessMixin` context),
+        and process them by kicking events over to the configured broker.
+
+        Raises:
+            SilverbackException: If there are no configured tasks to execute.
```

Comment:
```suggestion
            :class:`~silverback.exceptions.SilverbackException`: If there are no configured tasks to execute.
```

That being said, that should probably be a different error. You don't typically raise those base-class exceptions like that.

Reply:
Changed to `Halt` in fecb347